rbac/ledger_sync/inbound/listener.py

```diff
@@ -108,31 +104,30 @@ def listener():
     """ Listener for Sawtooth State changes
     """
     try:
-        database = Database()
-        database.connect()
+        conn = connect_to_db()
 
         LOGGER.info("Reading queued Sawtooth transactions")
         while True:
-            feed = database.run_query(database.get_table("inbound_queue"))
+            feed = r.table("inbound_queue").run(conn)
             count = 0
             for rec in feed:
-                process(rec, database)
+                process(rec, conn)
                 count = count + 1
             if count == 0:
                 break
             LOGGER.info("Processed %s records in the inbound queue", count)
         LOGGER.info("Listening for incoming Sawtooth transactions")
-        feed = database.run_query(database.get_table("inbound_queue").changes())
+        feed = r.table("inbound_queue").changes().run(conn)
         for rec in feed:
             if rec["new_val"] and not rec["old_val"]:  # only insertions
-                process(rec["new_val"], database)
+                process(rec["new_val"], conn)
 
     except Exception as err:  # pylint: disable=broad-except
         LOGGER.exception("Inbound listener %s exception", type(err).__name__)
         LOGGER.exception(err)
 
     finally:
         try:
-            database.disconnect()
+            conn.disconnect()
```

Comment:
There is no disconnect() function for rethinkdb that I am aware of.  There is a close() function, can you verify that this exists? or change it to close()?